utils/rpms/daos.spec

```diff
@@ -320,6 +320,8 @@ getent group daos_admins >/dev/null || groupadd -r daos_admins
 %config(noreplace) %{conf_dir}/daos_agent.yml
 %config(noreplace) %{conf_dir}/daos.yml
 %{_unitdir}/%{agent_svc_name}
+%{_mandir}/man8/daos.8*
+%{_mandir}/man8/dmg.8*
```

Comment:
Just as a double-check, these two commands are "client" node (only) commands, yes?

Reply:
I believe so. They are listed under the client files so it should be the client package only.